command/jsonprovider/block.go

```diff
@@ -12,8 +12,8 @@ type block struct {
 type blockType struct {
 	NestingMode string `json:"nesting_mode,omitempty"`
 	Block       *block `json:"block,omitempty"`
-	MinItems    uint64 `json:"min_items,omitempty"`
-	MaxItems    uint64 `json:"max_items,omitempty"`
+	MinItems    uint64 `json:"min_items"`
+	MaxItems    uint64 `json:"max_items"`
```

Comment:
For this one maybe we _should_ `omitempty` since zero here is considered to mean "no limit" and it'll make the output a little more compact in the common un-constrained case. :thinking: